app/src/main/res/drawable/ic_baseline_person_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="50dp" android:tint="?attr/colorControlNormal"
```

Comment:
I actually added this as an icon, maybe it can be reused. I've renamed it though.

Reply:
Yep, I can use it !